test/e2e/logs_multi_pipeline_max_pipeline_test.go

```diff
@@ -0,0 +1,92 @@
+//go:build e2e
+
+package e2e
+
+import (
+	"fmt"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	admissionregistrationv1 "k8s.io/api/admissionregistration/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	kitk8s "github.com/kyma-project/telemetry-manager/test/testkit/k8s"
+	kitkyma "github.com/kyma-project/telemetry-manager/test/testkit/kyma"
+	"github.com/kyma-project/telemetry-manager/test/testkit/periodic"
+	"github.com/kyma-project/telemetry-manager/test/testkit/suite"
+	"github.com/kyma-project/telemetry-manager/test/testkit/verifiers"
+)
+
+var _ = Describe(suite.ID(), Label(suite.LabelLogs), Ordered, func() {
+
+	Context("Before deploying a logpipeline", func() {
+		It("Should have a healthy webhook", func() {
+			verifiers.WebhookShouldBeHealthy(ctx, k8sClient)
+		})
+	})
+
+	Context("When reaching the pipeline limit", Ordered, func() {
+		const maxNumberOfLogPipelines = 5
+
+		var (
+			pipelinesNames = make([]string, 0, maxNumberOfLogPipelines)
+		)
+
+		makeResources := func() []client.Object {
+			var objs []client.Object
+			httpHostSecret := kitk8s.NewOpaqueSecret("log-rcv-hostname", kitkyma.DefaultNamespaceName,
+				kitk8s.WithStringData("log-host", "http://log-host:9880"))
+			objs = append(objs, httpHostSecret.K8sObject())
+			for i := 0; i < maxNumberOfLogPipelines; i++ {
+				pipelineName := fmt.Sprintf("%s-limit-%d", suite.ID(), i)
+				pipeline := kitk8s.NewLogPipelineV1Alpha1(pipelineName).
+					WithSecretKeyRef(httpHostSecret.SecretKeyRefV1Alpha1("log-host")).
+					WithHTTPOutput()
+				pipelinesNames = append(pipelinesNames, pipelineName)
+
+				objs = append(objs, pipeline.K8sObject())
+			}
+
+			return objs
+		}
+
+		BeforeAll(func() {
+
+			Eventually(func(g Gomega) {
+				var validatingWebhookConfiguration admissionregistrationv1.ValidatingWebhookConfiguration
+				g.Expect(k8sClient.Get(ctx, client.ObjectKey{Name: kitkyma.WebhookName}, &validatingWebhookConfiguration)).Should(Succeed())
+				g.Expect(validatingWebhookConfiguration.Webhooks).Should(HaveLen(2))
+			}, periodic.EventuallyTimeout, periodic.DefaultInterval).Should(Succeed())
```

Comment:
Why do we only have it in this test suite?

Reply:
in this test suite, we rely on webhook functionality